docs/docker.md

```diff
@@ -1,26 +1,265 @@
+```
+
+This will download the latest version of the `reaspeech-gpu` image from Docker
+Hub and run it in a container. The `--gpus all` flag tells Docker to give the
+container access to all GPUs on your machine. The `-p 9000:9000` flag maps port
+9000 on your local machine to port 9000 in the container, so you can access the
+ReaSpeech web interface at [http://localhost:9000](http://localhost:9000).
+
+## Setting up the ReaSpeech script
+
+Once the Docker container is running, you'll need to set up the ReaSpeech script
+in REAPER to communicate with the Docker container. Click the "9000:9000" link
+in the Docker Desktop interface or navigate to
+[http://localhost:9000](http://localhost:9000).
+You should see a web page that explains how to install and run ReaSpeech.
+The ReaSpeech script can be downloaded from this page.
+Before running the script, you will need to follow the instructions to ensure
+that ReaPack and ReaImGui are installed in REAPER.
+
+## Stopping the Docker Container
+
+To stop the Docker container, you'll need to know the container ID. You can get
+a list of running containers and their IDs by running:
 
 ```sh
-# Build and Run for CPU
-docker build -t reaspeech .
-docker run -d -p 9000:9000 --name reaspeech reaspeech
+docker ps
+```
+
+This will show you a list of running containers, along with their IDs, names,
+and other information. Find the ID of the ReaSpeech container you want to stop,
+and then run:
+
+```sh
+docker stop <container_id>
+```
+
+Replace `<container_id>` with the ID of the container you want to stop. This
+will stop the container, but it will still be on your machine. To remove the
+container completely, run:
+
+```sh
+docker rm <container_id>
+```
+
+This will remove the container from your machine.
+
+## Updating the Docker Image
+
+To update the Docker image to the latest version, you can pull the latest
+version from Docker Hub and run it in a new container. First, stop and remove
+the existing container:
+
+```sh
+docker stop <container_id>
+docker rm <container_id>
+```
+
+Then pull the latest version of the image and run it in a new container:
+
+```sh
+docker pull techaudiodoc/reaspeech:latest
+docker run -d -p 9000:9000 techaudiodoc/reaspeech:latest
+```
+
+This will download the latest version of the `reaspeech` image from Docker Hub
+and run it in a new container.
+
+## Troubleshooting
 
-# Build and Run for CPU (Development)
+If you're having trouble running the Docker image, here are a few things to
+check:
+
+- Make sure Docker is installed and running on your machine.
+- Make sure you're using the correct command to run the Docker image.
+- Check the Docker logs for the container to see if there are any error
+  messages.
+- If you're using the GPU version of the Docker image, make sure you have a
+  compatible NVIDIA GPU and that the NVIDIA container toolkit is installed on
+  your machine.
+
+If you're still having trouble, feel free to reach out to the ReaSpeech team
+for help.
```

Comment:
do we have a discord perma-invite we can toss in here?

Reply:
Probably, but we should mention it on the main README first. Going to wait on this until we're ready.